charts/apisix-ingress-controller/templates/configmap.yaml

```diff
@@ -18,24 +18,24 @@ apiVersion: v1
 data:
   config.yaml: |
     # log options
-    log_level: {{ .Values.ingressController.config.logLevel | quote }}
```

Comment:
why remove `ingressController` from yaml?
1. Reduce hierarchy
2. Easy to integrated into [apisix-helm-chart](https://github.com/apache/apisix-helm-chart) as subchart.

Reply:
Agree, please revert this change.